masonry/src/contexts.rs

```diff
@@ -1069,6 +1074,12 @@ impl_context_method!(LayoutCtx<'_>, PaintCtx<'_>, {
     }
 });
 
+impl PaintCtx<'_> {
+    pub fn debug_paint(&self) -> bool {
```

Comment:
I think when making this property public, ideally we have documentation about what it does roughly, or what this is for, as I don't think it's immediately obvious.

Reply:
I have added documentation in [7b21dda](https://github.com/linebender/xilem/pull/735/commits/7b21ddaceb9d2b0d9e9a46f54a656b6faff98623) 👍